setup/environment-linux.yml

```diff
@@ -77,4 +77,5 @@ dependencies:
     - websockets==9.1
     - yarl==1.4.2
     - git+https://github.com/CoinAlpha/python-signalr-client.git
+    - git+https://github.com/robotics-4-all/commlib-py.git@v2
```

Comment:
Should be
`git+https://github.com/robotics-4-all/commlib-py.git@f4d11c37f4aaaa20600e748fcb09bf3487f057d1`